src/Spice86/ViewModels/DebugWindowViewModel.cs

```diff
@@ -102,7 +103,7 @@ public DebugWindowViewModel(State state, Configuration configuration, Stack stac
         MemoryViewModel stackMemoryViewModel = new(memory, state,
             BreakpointsViewModel, pauseHandler, messenger,
             uiDispatcher, textClipboard, storageProvider, structureViewModelFactory,
-            canCloseTab: false, startAddress: stack.PhysicalAddress) {
+            canCloseTab: false, startAddress: MemoryUtils.ToSegmentedAddress(stack.PhysicalAddress)) {
```

Comment:
This needs the SS and SP register values, not the physical address in my opinion.

Reply:
This is MemoryUtils.ToPhysicalAddress(_state.SS, _state.SP) behind this property.